scala/private/phases/phase_merge_jars.bzl

```diff
@@ -3,17 +3,42 @@
 #
 # DOCUMENT THIS
 #
-load(
-    "@io_bazel_rules_scala//scala/private:rule_impls.bzl",
-    "merge_jars",
-)
 
 def phase_merge_jars(ctx, p):
```

Comment:
Same here. The reason I named this phase `phase_merge_jars` is because it's calling a function `merge_jars`, but they can be combined.

Reply:
fixed